<launch>

  <arg name="directory" default="$(env NIST_IMAGES)"/>
  <arg name="replayRate" default="10."/>
  
  <!-- Play files from folder -->
  <node name="ImagePublisher" pkg="imagePublisher" type="imagePublisher">
        <param name="imageFolderPath" value="$(arg directory)"/>
	<param name="replayRate"      value="$(arg replayRate)"/>	
	<param name="delayStart"      value="0"/>
  </node>

</launch>
